Tools/autotest/sim_vehicle.py

```diff
@@ -1031,6 +1059,18 @@ def generate_frame_help():
     print("May not use lldb with gdb")
     sys.exit(1)
 
+if cmd_opts.instance < 0:
+    print("May not specify a negative instance ID")
+    sys.exit(1)
+
+if cmd_opts.count < 1:
+    print("May not specify a count less than 1")
+    sys.exit(1)
+
+if cmd_opts.scrimmage and (cmd_opts.vehicle != 'ArduPlane' and cmd_opts.vehicle != 'ArduCopter'):
```

Comment:
Not in ["arduplane","arducopter"] maybe?

Reply:
Ok